cerberus/__init__.py

```diff
@@ -10,14 +10,17 @@
 
 from __future__ import absolute_import
 
-import pkg_resources
+from pkg_resources import get_distribution, DistributionNotFound
 
 from cerberus.validator import DocumentError, Validator
 from cerberus.schema import rules_set_registry, schema_registry, SchemaError
 from cerberus.utils import TypeDefinition
 
 
-__version__ = pkg_resources.get_distribution('Cerberus').version
+try:
+    __version__ = get_distribution(__name__).version
```

Comment:
I think using `__name__` here is too dynamic and it would break if this part is moved to anotehr module.

Reply:
I was wondering whether it could/should be more generic, because I didn't find a test which covers the case you describe. But I see your point and change it back again.